_includes/cwl/metadata_example.cwl

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env cwl-runner
+
+class: CommandLineTool
+id: Example tool
+label: Example tool
+cwlVersion: v1.0
+doc: |
+    An example tool demonstrating metadata.
+
+requirements:
+  - class: ShellCommandRequirement
+
+inputs:
+  bam_input:
+    type: File
+    doc: The BAM file used as input
+    format: http://edamontology.org/format_2572
+    inputBinding:
+      position: 1
+
+stdout: output.txt
+
+outputs:
+  report:
+    type: File
+    format: http://edamontology.org/format_1964
```

Comment:
For ORCIDs we shouldn't namespace them, see https://support.orcid.org/knowledgebase/articles/116780-structure-of-the-orcid-identifier "Storage of the ORCID iD in a database"
> When stored, the ORCID iD should be expressed as a URI: http://orcid.org/xxxx-xxxx-xxxx-xxxx, (with the protocol (http://), and with hyphens in the number xxxx-xxxx-xxxx-xxxx).